scripts/docker/rhel.6/Dockerfile

```diff
@@ -6,7 +6,9 @@
 # Dockerfile that creates a container suitable to build dotnet-cli
 FROM microsoft/dotnet-buildtools-prereqs:centos-6-d485f41-20173404063424
 
-RUN yum -q -y install sudo
+# yum doesn't work with the special curl version we have in the base docker image,
+# so we remove /usr/local/lib from the library path for this command
+RUN LD_LIBRARY_PATH= yum -q -y install sudo
```

Comment:
Is the extra space after the '=' a problem here? In a Windows command shell, right or wrong, the extra space would be preserved.

Reply:
The space after the `=` means we are assigning empty string to the variable.